docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,318 @@
+| `date-mday`       | `DD`                                                       | 4.3.4                | 01-28, 01-29, 01-30, 01-31 based on month/year                                                                  |
+| `time-hour`       | `hh`                                                       | 4.3.8a               | 00-23                                                                                                           |
+| `time-minute`     | `mm`                                                       | 4.3.9a               | 00-59                                                                                                           |
+| `time-second`     | `ss`                                                       | 4.3.10a              | 00-58, 00-59, 00-60 based on leap second rules                                                                  |
+| `time-secfrac`    | `s`                                                        | 5.3.14               | There must be one digit, but the max number of digits is implemenation defined                                  |
+| `time-numoffset`  | `("+" / "-") time-hour ":" time-minute`                    |                      |                                                                                                                 |
+| `time-offset`     | `"Z" / time-numoffset`                                     |                      |                                                                                                                 |
+| `partial-time`    | `time-hour ":" time-minute ":" time-second [time-secfrac]` |                      |                                                                                                                 |
+| `full-date`       | `date-fullyear "-" date-month "-" date-mday`               | 5.2.2.1b             | Extended calendar date                                                                                          |
+| `full-time`       | `partial-time time-offset`                                 | 5.3.3 or 5.3.4.2     | 5.3.3 is "UTC of day" while 5.3.4.2 is "Local time of day with the time shift between local time scale and UTC" |
+| `date-time`       | `full-date "T" full-time`                                  | 5.4.2.1              | Extended calendar date and time of day                                                                          |
+
+## Extended ISO 8601-1:2019 Profile in <xref:System.Text.Json>
+
+### Parsing
+
+The extended ISO 8601 profile implemented in System.Text.Json uses these components to define the following levels of date-time granularity:
+
+1. `full-date` (ISO 8601-1:2019 5.2.2.1)
+	1. `YYYY-MM-DD`
+
+ISO 8601-1:2019 5.2.2.2 "Representations with reduced precision" allows for just YYYY-MM (a) and just YYYY (b), but we currently don't permit it.
```

Comment:
"currently" makes sense in a code comment, but in official documentation it sounds like "and we're planning on it".

I'd either delete this sentence or replace it with a more declarative one:

```suggestion
ISO 8601-1:2019 5.2.2.2 "Representations with reduced precision" allows for just YYYY-MM (a) and just YYYY (b), but these representations are not supported.
```